assets/js/state/sapSystems.js

```diff
@@ -149,6 +149,15 @@ export const sapSystemsListSlice = createSlice({
         }
       );
     },
+    updateApplicationInstanceAbsentAt: (state, { payload: instance }) => {
+      const { absent_at } = instance;
```

Comment:
Could we have this like the `database` reducer?

Reply:
done